lib/internal/util.js

```diff
@@ -40,3 +42,42 @@ exports.deprecate = function(fn, msg) {
 
   return deprecated;
 };
+
+exports._errnoException = function(err, syscall, original) {
```

Comment:
Since this is going into an internal module, it doesn't need the underscore name.

Also, can we name the function (`exports.errnoException = function errnoException(..) {`) so we don't have to deal with `exports.errnoException()` below?